include/deal.II/base/vectorization.h

```diff
@@ -656,6 +656,16 @@ class VectorizedArray
     base_ptr[offsets[0]] = data;
   }
 
+  /**
+   * Returns sum over entries of data field.
```

Comment:
Can you please expand the text somewhat with the underlying algorithm, i.e.,
```suggestion
   * Returns sum over entries of the data field, $\sum_{i=1}^{\text{size}()} this->data[i]$.
```
Alternatively, you can also use a `@code` section as done a few lines up.

Reply:
Sure